src/angular/sdk-setup.ts

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+
+// @ts-expect-error - magicast is ESM and TS complains about that. It works though
+import { loadFile, ProxifiedModule, writeFile } from 'magicast';
+
+import * as path from 'path';
+
+// @ts-expect-error - clack is ESM and TS complains about that. It works though
+import clack from '@clack/prompts';
+import chalk from 'chalk';
+import { updateAppModuleMod } from './codemods/main';
+import { updateAppConfigMod } from './codemods/app-config';
+import type { SemVer } from 'semver';
+
+export function hasSentryContent(
+  fileName: string,
+  fileContent: string,
+  expectedContent = '@sentry/angular',
+): boolean {
+  const includesContent = fileContent.includes(expectedContent);
+
+  if (includesContent) {
+    clack.log.warn(
+      `File ${chalk.cyan(
+        path.basename(fileName),
+      )} already contains ${expectedContent}.
+Skipping adding Sentry functionality to ${chalk.cyan(
+        path.basename(fileName),
+      )}.`,
+    );
+  }
+
+  return includesContent;
+}
+
+export async function initalizeSentryOnAppModule(
+  dsn: string,
+  selectedFeatures: {
+    performance: boolean;
+    replay: boolean;
+  },
+): Promise<void> {
+  const appModuleFilename = 'main.ts';
+  const appModulePath = path.join(process.cwd(), 'src', appModuleFilename);
+
+  const originalAppModule = await loadFile(appModulePath);
+
+  if (hasSentryContent(appModulePath, originalAppModule.$code)) {
+    return;
+  }
+
+  const updatedAppModuleMod = updateAppModuleMod(
+    originalAppModule,
+    dsn,
+    selectedFeatures,
+  );
+
+  await writeFile(updatedAppModuleMod.$ast, appModulePath);
```

Comment:
m: I think we need a try/catch around this and gracefully abort the wizard. A single typo in user's config would crash this.